expenses/models.py

```diff
@@ -0,0 +1,54 @@
+from django.db import models
+
+
+class Expenses(models.Model):
+    # house_id = models.ForeignKey('user_id', on_delete=models.CASCADE)
```

Comment:
This line is in as a comment, till the house app PR will be approved

Reply:
I'll update you when #26 & #28 will be approved so you can remove the comment 👍